Atomic/syscontainers.py

```diff
@@ -2165,13 +2165,18 @@ def _pull_docker_tar(self, tarpath, image):
 
     def _skopeo_copy_img_to_ostree(self, img, skopeo_img_source, src_creds=None, insecure=None):
         """
-        Pefrom skopeo copy operation to copy images from img_source to ostree location based on image
+        Peform skopeo copy operation to copy images from img_source to ostree location based on image
```

Comment:
Perform :-)

Reply:
oops, ah, missed an r , thanks for pointing it out :)